### PR TITLE
PLT-2530 Fix team type not being set on some page loads in InviteMemberModal

### PR DESCRIPTION
**Pull request by jwilander**

### CODE REVIEW

File: webapp/components/invite_member_modal.jsx [hunk 1]
```diff
@@ -59,6 +59,8 @@ class InviteMemberModal extends React.Component {
         this.removeInviteFields = this.removeInviteFields.bind(this);
         this.showGetTeamInviteLinkModal = this.showGetTeamInviteLinkModal.bind(this);
 
+        const team = TeamStore.getCurrent();
```
Comment (crspeller) [inline webapp/components/invite_member_modal.jsx]: If this fails you will never get the team here.

Comment (jwilander) [inline webapp/components/invite_member_modal.jsx]: I'm not sure what you mean? There is already a listener for if the team changes after this step

Comment (crspeller) [inline webapp/components/invite_member_modal.jsx]: Ya your right.

**Comment by crspeller** [comment]
+1

**Pull request merged**